webpack.config.ts

```diff
@@ -64,6 +64,7 @@ function resolve(
 	// We rely on Buffer for our bridget thrift client
 	if (isClient) {
 		resolveOptions.alias = { ...resolveOptions.alias, Buffer: 'buffer' };
+		resolveOptions.fallback = { crypto: false };
```

Comment:
@JamieB-gu webpack stopped compiling client side bundles out of the blue, don't know why .. it was something to do with `image-rendering` and `crypto`.. but I don't understand it as nothing has changed 🤷‍♂️

Reply:
Something has changed, we've added a new component to the `import`s in the client-side bundle. And the import tree results in `image-rendering` being included client-side. I'd suggest breaking `FollowStatus` out into its own file rather than exporting it from `Follow`. That should narrow the import tree enough to exclude `image-rendering`.